hydra-node/src/Hydra/Chain/Direct/Tx.hs

```diff
@@ -78,6 +81,7 @@ import Hydra.Party (Party, partyFromChain, partyToChain)
 import Hydra.Plutus.Extras (posixFromUTCTime, posixToUTCTime)
 import Hydra.Plutus.Orphans ()
 import Hydra.Snapshot (Snapshot (..), SnapshotNumber, fromChainSnapshot)
+import Ouroboros.Consensus.Cardano.Block (StandardBabbage)
```

Comment:
Should: try to avoid hard-coding mentions of `Babbage` and also not increase density of the dependency graph

This import is only used in a type signature and can be avoided by using our type aliases of `hydra-cardano-api`.